honeybee_grasshopper_energy/src/HB Window Gap Material.py

```diff
@@ -30,10 +30,10 @@
 
 ghenv.Component.Name = "HB Window Gap Material"
 ghenv.Component.NickName = 'GapMat'
-ghenv.Component.Message = '0.1.0'
+ghenv.Component.Message = '0.1.1'
 ghenv.Component.Category = "Energy"
-ghenv.Component.SubCategory = "1 :: Construction"
-ghenv.Component.AdditionalHelpFromDocStrings = "5"
+ghenv.Component.SubCategory = "1 :: Constructions"
```

Comment:
and what was wrong with construction as is?

Reply:
I have been using plural names for all of the other sub-tabs and I want to keep things consistent:
0 :: Basic Properties
2 :: Schedules
3 :: Loads
I could also make all of them singular but I think plural makes more sense. I see that you also use plural names in the older HB[+].